### ISSUE TITLE
[MathSVG] external svg files

### ISSUE THREAD

**Issue by rschmehl** [rajgoel/reveal.js-plugins#17]
The plugin works fine for inline svg but it does not work for external svg, e.g. loaded via <img>. Maybe this is out of scope for this plugin, in any case, it would be nice to note this.

**Comment by rajgoel** [comment]
Thanks for reporting this. The plugin should also work for external svg, however, it appears that there is a problem with the synchronisation of tasks. Looks like the rendering takes place before the svg is loaded. Need to take a look at this.

**Comment by rschmehl** [comment]
Thank you. It would be great if this could be fixed because I have many slides with SVG figures containing fragment layers with Latex math content. Until now I ran an offline pdflatex compilation loop to convert the Latex into outlined svg object but this process breaks the SVG layers.

**Comment by rajgoel** [comment]
Yeah, I'll try my best, but it may take a bit before I can free some time for this

**Comment by rajgoel** [comment]
Just had a look at the plugin. As the plugin does not load the SVGs itself, this is not really something that the plugin has control over. A possible fix that I see is to make sure to initialise the plugin only after all SVGs are actually loaded.

**Comment by rschmehl** [comment]
Thanks. This makes sense to me. Could you give me a hint how I would initialise the plugin after all SVGs have been loaded?

**Comment by rajgoel** [comment]
Not sure how you can identify when all images have been loaded. However, you could also wait for a suitable amount of time using a timer. Then you could try adding this line

```
this.initialize = typesetMathInSVG;
```

just before the last line of the plugin and call ```RevealMathSVG.initialize()```. Haven't tested whether this works, this is just my first guess.
